sklearn/calibration.py

```diff
@@ -141,6 +141,9 @@ class CalibratedClassifierCV(ClassifierMixin,
     classes_ : ndarray of shape (n_classes,)
         The class labels.
 
+    n_features_in_ : int
+        Number of features seen during :term:`fit`.
```

Comment:
for `cv='prefit'`, this is only defined if `n_features_in_` is defined by `self.base_estimator`.